python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -155,16 +158,17 @@ def __init__(self):
         }
 
         self.q = PriorityQueue()
-        self.reconnect_failed = False 
+        self.disconnected = False 
         self.component = None
         self.cpe = ChillerPacketEncoder()
-
-        self.queue_task = None
-        self.watchdog_task = None
         self.watchdogLoopBool = False
```

Comment:
I think this name is against PEP8 convention. Should be `watchdog_loop_bool` although, I would recommend renaming it to something more intuitive like `run_watchdog`